docs/_source/getting_started/installation/deployments/huggingface-spaces.md

```diff
@@ -2,6 +2,10 @@
 
 Argilla nicely integrates with the Hugging Face stack (`datasets`, `transformers`, `hub`, and `setfit`), and now it can also be deployed using the Hub's Spaces.
 
+```{warning}
+**HuggingFace Spaces are reset** after 48 hours of inactivity and **do not have persistent storage**, so make sure to export your data after each annotation session to not lose your data. You can enable this via the Hugging Face Spaces settings.
```

Comment:
```suggestion
HuggingFace Spaces now have persistent storage and this is supported from Argilla 1.11.0 onwards, but you will need to manually activate it via the HuggingFace Spaces settings. Otherwise, unless you're on a paid space upgrade, after 48 hours of inactivity the space will be shut off and you will lose all the data. To avoid losing data, we highly recommend using the persistent storage layer offered by HuggingFace.
```